<?xml version="1.0" encoding="UTF-8"?>

<launch>
  
  <node pkg="rosbag" type="play" name="player" args="-l $(find object_labeling)/bag/yolo.bag" />
  
  <param name="point_cloud_topic" value="/hsrb/head_rgbd_sensor/depth_registered/rectified_points" />
  <param name="base_frame" value="base_link" />

  <node name="plane_segmentation_node" pkg="plane_segmentation" type="plane_segmentation_node" output="screen"/>

  <!-- Launch rviz -->
 	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find plane_segmentation)/config/config.rviz"/>
  
</launch>
